### PR TITLE
docs: add link to one of @manniL's videos on layers

### LINKED ISSUE #26953: Docs: Nuxt Layers video update with latest one
### Environment

Does not apply, as it is just update of documentation

### Reproduction

Docs Link: https://nuxt.com/docs/getting-started/layers

### Describe the bug

Existing video in layers documentation is 1 year old. I wanted it to be updated with latest video by Alexander Lichter.
[old video](https://www.youtube.com/watch?v=lnFCM7c9f7I)
[new video](https://www.youtube.com/watch?v=fr5yo3aVkfA&t=271s)

### Additional context

_No response_

### Logs

_No response_

### PR DESCRIPTION
**Pull request by pothuraju-dev**
### 🔗 Linked issue

resolves https://github.com/nuxt/nuxt/issues/26953

### 📚 Description

Documentation update

### CODE REVIEW

File: docs/1.getting-started/9.layers.md [hunk 1]
```diff
@@ -47,8 +47,8 @@ Nuxt uses [unjs/c12](https://c12.unjs.io) and [unjs/giget](https://giget.unjs.io
 Read more about layers in the **Layer Author Guide**.
 ::
 
-::tip{icon="i-ph-video-duotone" to="https://www.youtube.com/watch?v=lnFCM7c9f7I" target="_blank"}
-Watch Learn Vue video about Nuxt Layers.
+::tip{icon="i-ph-video-duotone" to="https://www.youtube.com/watch?v=fr5yo3aVkfA&t=271s" target="_blank"}
```
Comment (danielroe) [inline docs/1.getting-started/9.layers.md:50]: Maybe it would be worth linking to both videos?

Comment (pothuraju-dev) [reply docs/1.getting-started/9.layers.md:50]: Sure I will add both videos as two tips in docs

File: docs/1.getting-started/9.layers.md [hunk 2]
```diff
@@ -51,6 +51,10 @@ Read more about layers in the **Layer Author Guide**.
 Watch Learn Vue video about Nuxt Layers.
```
Comment (danielroe) [inline docs/1.getting-started/9.layers.md:51]:
```suggestion
Watch a video from Learn Vue about Nuxt Layers.
```

File: docs/1.getting-started/9.layers.md [hunk 3]
```diff
@@ -51,6 +51,10 @@ Read more about layers in the **Layer Author Guide**.
...
 ::
 
+::tip{icon="i-ph-video-duotone" to="https://www.youtube.com/watch?v=fr5yo3aVkfA&t=271s" target="_blank"}
+Watch Alexander Lichter video about Nuxt Layers.
```
Comment (danielroe) [inline docs/1.getting-started/9.layers.md:55]:
```suggestion
Watch a video from Alexander Lichter about Nuxt Layers.
```

**Review by danielroe — APPROVED** [review]
Thank you!

cc: @manniL

**Review by TheAlexLichter — APPROVED** [review]

**Pull request merged**